### PR TITLE
header functionality

### PR DESCRIPTION
**Pull request by owner6**

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+	pos = Math.max(pos -1, 0) 
+	setTransform()
+}
+
+function next() {
+	pos = Math.min(pos + 1, itemCount - 1)
+	setTransform()
+}
+
+window.addEventListener('resize', setTransform)
```
Comment (dever4eg) [inline script.js]: You forgot to add an empty line

File: script.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
+let d = document;
```
Comment (dever4eg) [inline script.js]: I suggest to rename this file to `adaptive-header.js` because it is part of adaptive-header page

File: adaptive-header.html [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+				</div>
+				<button onclick="next()">&raquo;</button>
+
+			</div>
+
+		</div>
+	</main>
+
+	<script src="https://cdnjs.cloudflare.com/ajax/libs/modernizr/2.8.3/modernizr.min.js" type="text/javascript"></script>
+	<script src="script.js"></script>
```
Comment (dever4eg) [inline adaptive-header.html]: And do not forget to change file name here too

**Review by dever4eg — CHANGES_REQUESTED** [review]

File: script.js (hunk 1 above)
Comment (owner6) [reply script.js]: corrected

File: script.js (hunk 2 above)
Comment (owner6) [reply script.js]: thanks, renaimed file

File: adaptive-header.html (hunk 3 above)
Comment (owner6) [reply adaptive-header.html]: thanks, recorrected

**Review by dever4eg — APPROVED** [review]

**Pull request merged**